packages/alchemy/src/middleware/client.ts

```diff
@@ -1,4 +1,5 @@
 import {
+  type Percentage,
```

Comment:
nit: might need to lint this file. 
![image](https://github.com/alchemyplatform/aa-sdk/assets/43521356/915c8a11-b147-4a97-9cee-d5bee04b8deb)